src/sentry/utils/migrations.py

```diff
@@ -0,0 +1,14 @@
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def clear_flag(Model, flag_name, flag_attr_name="flags"):
+    """
+    This function is used to clear an existing flag value for all items in a given model
+    """
+    for item in RangeQuerySetWrapperWithProgressBar(Model.objects.all()):
+        flags = getattr(item, flag_attr_name)
+        if flags[flag_name]:
+            # clear the flag
+            new_flag_value = flags & ~(getattr(Model, flag_attr_name)[flag_name])
+            setattr(item, flag_attr_name, new_flag_value)
```

Comment:
Can we do `setattr(item, flag_attr_name, False)` here?

Reply:
@wedamija wouldn't that just do something like
```
Model.flags = False
```

Reply:
Right, I think I mean
`getattr(item, flag_attr_name)[flag_name] = False`

Reply:
If so, that def won't work lol

Reply:
@wedamija switching to `.filter().update()` so I'm using a different way of doing it